styles/mapillary-js.css

```diff
@@ -29,7 +29,7 @@
     right: 0;
     bottom: 0;
     left: 0;
-    z-index: 10;
+    z-index: -1;
```

Comment:
Feels like the DetectionComponent styles should be in a DetectionComponent.css file and not in the general mapillary-js css file.